staging/publishing/rules.yaml

```diff
@@ -845,31 +1051,69 @@ rules:
       branch: release-1.13
     - repository: client-go
       branch: release-10.0
+  - source:
+      branch: release-1.14
+      dir: staging/src/k8s.io/cloud-provider
+    name: release-1.14
+    go: 1.11.5
+    dependencies:
+      - repository: api
+        branch: release-1.14
+      - repository: apimachinery
+        branch: release-1.14
+      - repository: client-go
+        branch: release-11.0
 - destination: node-api
   library: true
   branches:
   - source:
       branch: master
       dir: staging/src/k8s.io/node-api
     name: master
+    go: 1.11.5
```

Comment:
We have the default go version defined here - https://github.com/kubernetes/publishing-bot/blob/master/pkg/golang/install.go#L32 so I don't think we need this.